compiler/aarch64/codegen/OMRMemoryReference.cpp

```diff
@@ -153,7 +155,8 @@ OMR::ARM64::MemoryReference::MemoryReference(
    _unresolvedSnippet(NULL),
    _flag(0),
    _scale(0),
-   _offset(disp)
+   _offset(disp),
```

Comment:
The type of `_offset` is changed from `int32_t` to `intptr_t`.
Perhaps you also want to change the type of `disp`.

Reply:
Changed the type of `disp` to `intptr_t`.